opentelekomcloud/acceptance/cce/resource_opentelekomcloud_cce_node_v3_test.go

```diff
@@ -16,36 +16,37 @@ import (
 
 var privateIP = "192.168.1.13"
 
-func TestAccCCENodesV3_basic(t *testing.T) {
+const nodeName = "opentelekomcloud_cce_node_v3.node_1"
```

Comment:
That's not exactly a Node name, rather a resource name

Reply:
I've renamed it to `resourceNameNode`, because const `resourceName` has collisions with the same const name in `resource_opentelekomcloud_cce_cluster_v3_test`